### ISSUE TITLE
Terms are not underlined everywhere

### ISSUE THREAD

**Issue by laulaz** [collective/collective.glossary#47]
Terms are not underlined (at least) on :
- mosaic pages
- descriptions / titles

**Comment by hvelarde** [comment]
yes, that's by design: only terms inside #content-core are underlined:

https://github.com/collective/collective.glossary/blob/1.0b1/src/collective/glossary/browser/static/main.js#L5

**Comment by laulaz** [comment]
@hvelarde you mean that it should not be improved (IOW it's a delibarate choice) ?  I can understand for title and description (for style and other reasons), but we could maybe do something for Mosaic composed pages ... what do you think ?

**Comment by hvelarde** [comment]
yes, if could be a configuration option; feel free to provide a pull request for that.

**Comment by laulaz** [comment]
There are 2 problems to solve to enable terms tooltips on Mosaic composed pages :
1. the ID inside which the terms are searched is defined to `#content-core` as @hvelarde mentioned, and should be changed / configurable ;
2. the code that checks if we are on a view (to enable tooltip JS) returns False for a Mosaic page (`@@page-site-layout`) : https://github.com/collective/collective.glossary/blob/101a5f58d73d46ddda61ece881d0878b912e3def/src/collective/glossary/browser/views.py#L103
